root/static/lib/leaflet/leaflet.markercluster-src.js

```diff
@@ -0,0 +1,2616 @@
+/*

```

Comment:
BTW, you can add it to package.json automatically like `npm install --save leaflet.markercluster`. But then you have to edit package.json manually and remove the caret from the version, e.g. if it has "^1.0.0" (which matches any 1.x.x release), change it to "1.0.0", because we prefer exact versions.